python/benchmarks/finalize_staged_data.py

```diff
@@ -74,32 +74,36 @@ def peakmem_finalize_staged_data(self, cache:CachedDFGenerator, param:int):
     def teardown(self, cache:CachedDFGenerator, param:int):
         self.ac.delete_library(self.lib_name)
 
-class FinalizeStagedDataWiderDataframeX3(FinalizeStagedData):
-    '''
-        The test is meant to be executed with 3 times wider dataframe than the base test
-    '''
+if sys.version_info >= (3, 8):
```

Comment:
I know this is from #2060 but since we upgrade the benchmarks to 3.11 we can remove this check?